src/components/journeys/JourneyInstancesDataTable/getStaticColumns.tsx

```diff
@@ -15,6 +15,14 @@ const getPeopleString = (people: ZetkinPersonType[]) =>
   people.map((person) => `${person.first_name} ${person.last_name}`).join(', ');
 
 export const getStaticColumns = (): GridColDef[] => [
+  {
+    field: 'id',
+    renderCell: (params) => {
+      const row = params.row as ZetkinJourneyInstance;
+      return row.id.toString();
```

Comment:
I'm thinking it would be a slightly nicer UX if the `#` prefix was added to the output, to correlate with what the user must search for.

Reply:
yeah good shout